web/docs/operators/azure-log-analytics.md

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_custom_props:
+  operator:
+    source: false
+    sink: true
+---
+
+# azure-log-analytics
+
+Sends events via the [Microsoft Azure Logs Ingestion
+API](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/logs-ingestion-api-overview).
+
+## Synopsis
+
+```
+azure-log-analytics --tenant-id <tenant-id> --client-id <client-id>
```

Comment:
Can you split the synopsis into multiple lines for legibility?

Reply:
Done in https://github.com/tenzir/tenzir/pull/4281/commits/0978fc38aa2668e38bb41980a7ac94c8ba7b491d